packages/wrangler/src/index.tsx

```diff
@@ -682,6 +719,11 @@ function createCLIParser(argv: string[]) {
               )}`
             );
 
+            await installPackages(
+              shouldRunPackageManagerInstall,
+              devDepsToInstall
+            );
+
```

Comment:
I understand that you are running this here before the `writePackageJsonScriptsAndUpdateWranglerToml()` function because you want the install to appear before the final instructions to the user.

But I think we could make this cleaner by having an extra "global" called `instructions: string[]`. Then `writePackageJsonScriptsAndUpdateWranglerToml()` could add the instructions to that list rather than logging them directly.

This would mean we would just need a single call to `installPackages()` write at the end, followed by a call to a function like `printInstructions()` which will then log out the instructions.